contents/tutorials/fake-door-test.md

```diff
@@ -0,0 +1,139 @@
+
+```bash
+npx create-next-app@latest fake-door
+```
+
+After doing this, go into your newly created `fake-door` folder and install `posthog-js`.
+
+```bash
+cd fake-door
+npm i posthog-js
+```
+
+After installing `posthog-js`, set up the PostHog initialization. This requires going to our `app` folder, creating a `providers.js` file, adding our PostHog initialization behind a window check with `opt_in_site_apps: true` (for later), and creating a provider component like this:
+
+```js-web
+// app/providers.js
+'use client'
+import posthog from 'posthog-js'
+import { PostHogProvider } from 'posthog-js/react'
+
+if (typeof window !== 'undefined') {
+  posthog.init('<ph_project_api_key>', {
+    api_host: '<ph_instance_address>',
+    opt_in_site_apps: true
+  })
+}
+
+export function PHProvider({ children }) {
+  return <PostHogProvider client={posthog}>{children}</PostHogProvider>
+}
+```
+
+With our `PHProvider` component created, we can import it and wrap our main app in it in `layout.js`.
+
+```js-web
+// app/layout.js
+import './globals.css'
+import { PHProvider } from './providers'
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang="en">
+      <PHProvider>
+        <body>{children}</body>
+      </PHProvider>
+    </html>
+  )
+}
+```
+
+With this done, PostHog starts autocapturing events and our app is ready for a fake door test.
+
+## Creating our fake door test
+
+Creating a fake door test is intentionally simple. There are many ways to do it, but we start with a basic one. Add a button to our app that says "Try our new feature."
+
+```js-web
+// app/page.js
+
+export default function Home() {
+  return (
+    <div>
+      <h1>Welcome to our app!</h1>
+      <button>Sign in</button>
+      <span> - </span>
+      <button>Try our new feature</button>
+    </div>
+  )
+}
+```
+
+Once this is set up, we can run `npm run dev` to run the development server, go to our app, and click the button a few times. This captures events in PostHog.
+
+Once we capture that data in PostHog, we can set up an action for our fake door test. To do this: 
+
+1. Go to the data management, click the action tab, and then click "[New action](https://app.posthog.com/data-management/actions/new)." 
+2. Either use the toolbar to select your element or select from an event (which we choose).
+3. Name your action "shown new feature interest." 
+4. Select "Autocapture" as our "Match Group #1," and then match the element text "Try our new feature" exactly. 
+5. Press save.
+
+![Action](../images/tutorials/fake-door-test/action.png)
+
+With this, you can keep track of validation on your fake door test and even send [webhook notifications](/docs/webhooks) when someone clicks the button. 
+
+## Adding a survey to our fake door test
+
+Since the goal of a fake door test is to validate interest, we can improve this a lot. Specifically, we can take users to another page and ask them a survey question.
```

Comment:
```suggestion
Since the goal of a fake door test is to validate interest, we can get even more feedback by adding a survey.

We'll implement this in our example by navigating the user to a new page when they click the button and asking them a survey question.
```